From 70de29171b31c1fa3c2ee0e8f21940f2dc3e8a9e Mon Sep 17 00:00:00 2001
From: Lars Randers <lranders@mail.dk>
Date: Wed, 16 Jul 2025 20:11:52 +0000
Subject: [PATCH] gpio: gpio-mpfs: add registration string

Signed-off-by: Lars Randers <lranders@mail.dk>
---
 drivers/gpio/gpio-mpfs.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpio/gpio-mpfs.c b/drivers/gpio/gpio-mpfs.c
index 8b5a6d76c..98d520144 100644
--- a/drivers/gpio/gpio-mpfs.c
+++ b/drivers/gpio/gpio-mpfs.c
@@ -278,6 +278,8 @@ static int mpfs_gpio_probe(struct platform_device *pdev)
 		girq->default_type = IRQ_TYPE_NONE;
 	}
 
+	dev_info(dev, "Registered MPFS GPIO controller\n");
+
 	return devm_gpiochip_add_data(dev, &mpfs_gpio->gc, mpfs_gpio);
 }
 
-- 
2.39.2

